### PR TITLE
refactor: cleanup JSON-LD contexts

### LINKED ISSUE #2751: Cleanup JSON-LD contexts
* replace https://trustbloc.github.io/context/vc/credentials-v1.jsonld context with https://w3id.org/security/jws/v1
* find the way to not use https://trustbloc.github.io/context/vc/examples-v1.jsonld in tests
* split test contexts from the global framework contexts

### PR DESCRIPTION
**Pull request by aholovko**
This PR:
* simplifies new JSON-LD document loader:
  - support for `WithContextFS` was removed (in practice it didn't justify itself)
  - "core" set of embedded contexts was defined (extra contexts can be preloaded using `WithExtraContexts`)
* replaces the following contexts:
  - `https://trustbloc.github.io/context/vc/credentials-v1.jsonld` -> `https://w3id.org/security/jws/v1`
  - `https://trustbloc.github.io/context/vc/examples-v1.jsonld` -> `https://w3id.org/vc-revocation-list-2020/v1` (only in BDD tests - this allows to remove it from the main embedded set)

Closes #2751

Signed-off-by: Andriy Holovko <andriy.holovko@gmail.com>

### CODE REVIEW

**Comment by troyronda** [comment]
@aholovko Please keep third_party above the domain name folders.

e.g., contexts/third_party/domain

**Comment by troyronda** [comment]
@aholovko Can you document how downstream projects should embed their additional contexts?

File: pkg/internal/jsonldtest/contexts/odrl.jsonld [hunk 1]
```diff
@@ -0,0 +1,200 @@
+{
```
Comment (troyronda) [inline pkg/internal/jsonldtest/contexts/odrl.jsonld:1]: should use contexts/third_party/<domain>/file convention for these files too.

**Comment by troyronda** [comment]
pkg/internal/jsonldtest/contexts/trustbloc-examples_v1.jsonld should be removed.

**Review by troyronda — APPROVED** [review]

**Comment by aholovko** [comment]
> pkg/internal/jsonldtest/contexts/trustbloc-examples_v1.jsonld should be removed.

Will be handled by https://github.com/hyperledger/aries-framework-go/issues/2761

**Pull request merged**